internal/dynamicanalysis/analysis.go

```diff
@@ -45,11 +67,58 @@ func Run(sb sandbox.Sandbox, args []string) (*Result, error) {
 	// Run the command
 	log.Debug("Running dynamic analysis command",
 		"args", args)
+
+	log.Debug("Reroute all http traffic through sslsplit")
+	iptables := exec.Command("iptables", "-t", "nat", "-A", "PREROUTING", "-i", "cni-analysis", "-p", "tcp", "--dport", "80", "-j", "REDIRECT", "--to-port", "8081")
+	err := iptables.Start()
+
+	if err != nil {
+		log.Fatal(err.Error())
+	}
+	err = iptables.Wait()
+	if err != nil {
+		log.Fatal(err.Error())
+	}
+
+	log.Debug("Reroute all https traffic through sslsplit")
+	iptables = exec.Command("iptables", "-t", "nat", "-A", "PREROUTING", "-i", "cni-analysis", "-p", "tcp", "--dport", "443", "-j", "REDIRECT", "--to-port", "8080")
+	err = iptables.Start()
+
+	if err != nil {
+		log.Fatal(err.Error())
+	}
+	err = iptables.Wait()
+	if err != nil {
+		log.Fatal(err.Error())
+	}
+
+	log.Debug("starting sslsplit")
+	sslsplit := exec.Command("sslsplit", "-d", "-L", "/tmp/ssl.flow", "-l", "/tmp/sslLinks.flow", "-k", "/proxy/certs/ca.pem", "-c", "/proxy/certs/ca.crt", "http", "0.0.0.0", "8081", "https", "0.0.0.0", "8080")
```

Comment:
The log files "ssl.flow" and "sslLinks.flow" linger between analysis runs.

This means that:
- the URLs from each phase are mixed up with the next phase
- the URLs from one package are mixed up with subsequent packages being analyzed.

These files should:
1. be uniquely named per `Run` invocation
1. be cleaned up after being parsed